tests/falco/commands_test.go

```diff
@@ -132,3 +134,38 @@ func TestFalco_Cmd_PluginInfo(t *testing.T) {
 			`No suggested open params available.*`),
 		res.Stdout())
 }
+
+func TestFalco_Print_IgnoredEvents(t *testing.T) {
+	t.Parallel()
+	checkDefaultConfig(t)
+	type Data struct {
+		Events []string `json:"events"`
+	}
+	evntfile, err := os.Open("events.json")
+	if err != nil {
+		panic(err)
+	}
+	defer evntfile.Close()
+
+	var events Data
+	err = json.NewDecoder(evntfile).Decode(&events)
+	if err != nil {
+		panic(err)
+	}
+
+	runner := tests.NewFalcoExecutableRunner(t)
+	t.Run("all-events-ignored-by-default", func(t *testing.T) {
```

Comment:
Since we don't have other tests here, I would avoid having a sub-test with `t.Run` and would just proceed with the test code in the main `TestFalco_Print_IgnoredEvents` test body, what do you think?

Reply:
>Test the behaviour and expected output of the Falco CLI depending on its options, considered both individually and in combinations:

Yes, since there no combinations for `-i` we can proceed with the main `TestFalco_Print_IgnoredEvents`